rules/do-you-gather-more-personal-information-progressively/rule.md

```diff
@@ -3,22 +3,32 @@ type: rule
 archivedreason: 
 title: Do you gather more personal information progressively?
 guid: f9928c70-a007-4d02-972d-46fdde6db4db
-uri: do-you-gather-more-personal-information-progressively
+uri: gather-personal-information-progressively
 created: 2015-02-13T02:50:56.0000000Z
-authors: []
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
 related: []
 redirects: []
```

Comment:
Hi @bettybondoc 

- [ ] 1. We need to add the old URI here for redirect
- [ ] 2. Also need to update the folder name to be consistent with the new URI
- [ ] 3. Also need to update the category(s) where this rule is with the new URI

Reply:
> 1. We need to add the old URI here for redirect
> 2. Also need to update the folder name to be consistent with the new URI

All Done - this PR

> 3. Also need to update the category(s) where this rule is with the new URI

Done - see https://github.com/SSWConsulting/SSW.Rules.Content/pull/8071